thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/api/user/dashboard/UserDashboardResource.java

```diff
@@ -345,7 +370,17 @@ public int compare(MergedAnomalyResultDTO o1, MergedAnomalyResultDTO o2) {
     return output;
   }
 
+  private Collection<? extends MergedAnomalyResultDTO> fetchFrameworkAnomaliesByMetricDataset(
+      List<Predicate> predicates) {
+    predicates.add(Predicate.NEQ("detectionConfigId", 0));
```

Comment:
don't need to add this predicate

Reply:
Anomalies is a list. This is a safety condition that avoids potential duplicates.